acceptance/support-skeletons/disk_dvd/Vagrantfile

```diff
@@ -0,0 +1,5 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "box"
+
+  config.vm.disk :disk, type: :dvd, name: "dvd1", file: "./example.iso"
```

Comment:
What's inside the example.iso? :smile:

Reply:
Still curious about what's inside these isos :smile: @jbonhag

Reply:
Nothing too sinister 😄

```
$ ls /Volumes/CDROM
README.md

$ cat /Volumes/CDROM/README.md
# README

This is a minimal ISO for testing purposes.
```